tfhe/src/integer/server_key/radix_parallel/tests_cases_comparisons.rs

```diff
@@ -1570,144 +2240,129 @@ macro_rules! define_signed_scalar_comparison_test_functions {
             {
+                #[cfg(not(feature = "__coverage"))]
                 PARAM_MESSAGE_2_CARRY_2_KS_PBS,
                 // We don't use PARAM_MESSAGE_3_CARRY_3_KS_PBS,
                 // as default test might overflow values
                 // and when using 3_3 to represent 128 we actually have more than 128 bits
                 // of message so the overflow behaviour is not the same, leading to false negatives
-                PARAM_MESSAGE_4_CARRY_4_KS_PBS
+                #[cfg(not(feature = "__coverage"))]
+                PARAM_MESSAGE_4_CARRY_4_KS_PBS,
+                #[cfg(feature = "__coverage")]
+                COVERAGE_PARAM_MESSAGE_2_CARRY_2_KS_PBS
             });
         }
     };
 }
 
-define_signed_scalar_comparison_test_functions!(eq);
-define_signed_scalar_comparison_test_functions!(ne);
-define_signed_scalar_comparison_test_functions!(lt);
-define_signed_scalar_comparison_test_functions!(le);
-define_signed_scalar_comparison_test_functions!(gt);
-define_signed_scalar_comparison_test_functions!(ge);
-
-fn integer_signed_unchecked_scalar_min_parallelized_128_bits(
-    params: crate::shortint::ClassicPBSParameters,
-) {
-    test_signed_unchecked_scalar_function(
-        params,
-        2,
-        ServerKey::unchecked_scalar_min_parallelized,
-        std::cmp::min,
-    );
-}
-
-fn integer_signed_unchecked_scalar_max_parallelized_128_bits(
-    params: crate::shortint::ClassicPBSParameters,
-) {
-    test_signed_unchecked_scalar_function(
-        params,
-        2,
-        ServerKey::unchecked_scalar_max_parallelized,
-        std::cmp::max,
-    );
-}
-
-fn integer_signed_smart_scalar_min_parallelized_128_bits(
-    params: crate::shortint::ClassicPBSParameters,
-) {
-    test_signed_smart_scalar_function(
-        params,
-        2,
-        ServerKey::smart_scalar_min_parallelized,
-        std::cmp::min,
-    );
-}
-
-fn integer_signed_smart_scalar_max_parallelized_128_bits(
-    params: crate::shortint::ClassicPBSParameters,
-) {
-    test_signed_smart_scalar_function(
-        params,
-        2,
-        ServerKey::smart_scalar_max_parallelized,
-        std::cmp::max,
-    );
-}
-
-fn integer_signed_scalar_min_parallelized_128_bits(params: crate::shortint::ClassicPBSParameters) {
-    test_signed_default_scalar_function(
-        params,
-        2,
-        ServerKey::scalar_min_parallelized,
-        std::cmp::min,
-    );
-}
-
-fn integer_signed_scalar_max_parallelized_128_bits(params: crate::shortint::ClassicPBSParameters) {
-    test_signed_default_scalar_function(
-        params,
-        2,
-        ServerKey::scalar_max_parallelized,
-        std::cmp::max,
-    );
-}
-
-create_parametrized_test!(integer_signed_unchecked_scalar_max_parallelized_128_bits {
+#[cfg(not(feature = "__coverage"))]
+define_signed_scalar_comparison_test_functions!(eq, i128);
+#[cfg(feature = "__coverage")]
```

Comment:
no need for feature gate here I think ? worst case make the functions pub if clippy complains they are only enabled for testing and indicate only the i128 are used for regular testing and i8 only are use for coverage